src/test/java/seedu/task/logic/LogicManagerTest.java

```diff
@@ -394,6 +394,31 @@ public void execute_select_jumpsToCorrectTask() throws Exception {
     }
 
     @Test
+    public void execute_MarkInvalidArgsFormat_errorMessageShown() throws Exception {
```

Comment:
sry what do u mean? there is a merge conflict with this branch and the master?
I don't see any merge conflicts tho?